react/Viewer/Toolbar.jsx

```diff
@@ -22,10 +22,10 @@ const Toolbar = ({ hidden, onMouseEnter, onMouseLeave, file, onClose, t }) => {
 
   return (
     <div
+      id="viewer-toolbar"
       className={cx(styles['viewer-toolbar'], {
         [styles['viewer-toolbar--hidden']]: hidden
       })}
-      role="viewer-toolbar"
```

Comment:
If we have to access the DOM node, we should use a ref instead of targetting by id. Is the id really necessary ?

Reply:
Le besoin est d'accéder à l'élément depuis l'app, et non dans UI. C'est possible avec une ref ?

Reply:
Oui c'est possible avec une ref.

Reply:
Je n'ai pas réussi à faire remonter une ref de `Toolbar` jusqu'à l'index du `Viewer` en passant par `ViewerControls`, pour ensuite la faire redescendre dans `FooterContent`. Par manque de temps je vais pour l'instant rester sur la solution de l'`id` et une `useMemo` côté app.